docs/gallery.rst

```diff
@@ -172,6 +172,23 @@ Examples that make use of the :ref:`contrib` module.
 
     <div class="sphx-glr-thumbnails">
 
+.. raw:: html
+
+    <div class="sphx-glr-thumbcontainer" tooltip="Adversarial training of CNN on MNIST.">
+
+.. only:: html
+
+  .. image:: /images/favicon.svg
```

Comment:
It would be great if you can use an example-specific image, such as this one instead of the generic favicon.svg one (see for example how it's done in the "Optimistic Gradient Descent in a Bilinear Min-Max Problem" example)

![image](https://github.com/google-deepmind/optax/assets/277639/9b95a86c-ca5d-4196-9718-7130e5197a5d)

Reply:
Having a nice looking thumbnail goes a long way in making the gallery pretty and the example easy to find :-)

Reply:
Happy New Year ❄🎉. Sure, I agree. Will add this image as a thumbnail. I was thinking maybe we could create a thumbnail template for the example gallery. Tell me what you think. If you agree, I can work on some sample designs.